controllers/iamrole_controller.go

```diff
@@ -227,6 +211,25 @@ func (r *IamroleReconciler) HandleReconcile(ctx context.Context, req ctrl.Reques
 		}
 		fallthrough
 
+	case iammanagerv1alpha1.Error:
```

Comment:
Add `Error` case after `Ready` to handle if iamrole is not created at AWS, iam-manager should not retrieve it.